src/AppRoutes.js

```diff
@@ -7,14 +7,14 @@ import PropTypes from 'prop-types';
 import { TabLayout } from './layouts';
 import { SignIn as SignInView, AccessDenied as AccessDeniedView } from './views';
 import Workspaces from './views/Workspaces';
-import { useWorkspacesList } from './state/hooks/WorkspaceHooks';
+import { useWorkspaceData } from './state/hooks/WorkspaceHooks';
 
 const AppRoutes = (props) => {
   const { authenticated, authorized, tabs } = props;
   const location = useLocation();
   const providedUrl = sessionStorage.getItem('providedUrl');
   const providedUrlBeforeSignIn = sessionStorage.getItem('providedUrlBeforeSignIn');
-  const workspacesList = useWorkspacesList();
+  const currentWorkspaceData = useWorkspaceData();
```

Comment:
it's seems that you use only the id of workspace, maybe you can use the useWorkspaceId hook instead